static/app/components/searchQueryBuilder/tokens/filter/utils.tsx

```diff
@@ -68,3 +68,28 @@ export function formatFilterValue(token: TokenResult<Token.FILTER>['value']): st
       return token.text;
   }
 }
+
+/**
+ * Replaces the focused parametr (at cursorPosition) with the new value.
+ *
+ * Example:
+ * replaceCommaSeparatedValue('foo,bar,baz', 5, 'new') => 'foo,new,baz'
```

Comment:
This is an awesome doc string 📝, the example helps a lot.

Thought two things:

1. Typo: `parametr`
2. Is the `cursorPosition` falling back to the end of the string standard for other of these helpers? If not it might help to add it to the doc string.

Reply:
Fixed the typo and added info about the fallback